app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] [REQUIRED] Good job so far! The associations between models should be set up correctly according, please implement `:author` correctly as below 🙏🏼

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```